templates/layouts/components/recent-area.php

```diff
@@ -0,0 +1,47 @@
+<div class="recent-post-area">
+                <h6>recent post</h6>
+                <div class="image-area-section">
+                    <div class="image-area">
+                        <img src="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/3-60x60.jpg" alt="">
+                    </div>    
+                    <div class="new-trend-area">
+                        <a class="post-title" href="http://wp.efforttech.net/newwp/sigma/2019/01/major-trends-expected-2018-4/">new trends</a>
+                    <div class="post-details">
+                        <a class="post-author" href="http://wp.efforttech.net/newwp/sigma/author/admin/">by admin</a>
+                    <div class="date-area">   
+                        <a class="post-date" href="http://wp.efforttech.net/newwp/sigma/21/01/">21 Jan</a>
```

Comment:
Change the href to `#`

Reply:
okay. All the href?

Reply:
all the href value where its an external link change `href="#"`